data/registry_model/registry_proxy_model.py

```diff
@@ -171,7 +232,7 @@ def lookup_manifest_by_digest(
                 manifest_digest,
                 self._create_manifest_with_temp_tag,
             )
-        except ManifestDoesNotExist as e:
+        except (ManifestDoesNotExist, QuotaExceededException) as e:
             raise e
```

Comment:
could you remove the `e` from this line as well? we don't want to mess with the stacktrace.

Reply:
done